Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/DrawTool.java

```diff
@@ -73,6 +78,12 @@ public boolean handleDown(PointF coordinate) {
 		mPreviousEventCoordinate = new PointF(coordinate.x, coordinate.y);
 		pathToDraw.moveTo(coordinate.x, coordinate.y);
 		movedDistance.set(0, 0);
+		pathInsideBitmap = false;
+
+		if ((coordinate.x < PaintroidApplication.drawingSurface.getBitmapWidth()) && (coordinate.y < PaintroidApplication.drawingSurface
```

Comment:
can you write a utility function for this check since it occurs more often than once?